content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -103,97 +97,60 @@ dotnet run
 
 ## Instrumentation
 
-Next, you'll use a [OpenTelemetry .NET Automatic Instrumentation](../automatic)
-to instrument the application at launch time. While you can [configure .NET
-Automatic Instrumentation][] in a number of ways, the steps below use Unix-shell
-or PowerShell scripts.
-
-> **Note**: PowerShell commands require elevated (administrator) privileges.
+Next we'll install the instrumentation
+[NuGet packages from OpenTelemetry](https://www.nuget.org/profiles/OpenTelemetry)
+that will generate the telemetry, and set them up.
 
-1. Download installation scripts from [Releases][] of the
-   `opentelemetry-dotnet-instrumentation` repository:
-
-   {{< tabpane text=true >}} {{% tab Unix-shell %}}
-
-   ```sh
-   curl -L -O https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest/download/otel-dotnet-auto-install.sh
-   ```
+1. Add the packages
 
-   {{% /tab %}} {{% tab PowerShell - Windows %}}
-
-   ```powershell
-   $module_url = "https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest/download/OpenTelemetry.DotNet.Auto.psm1"
-   $download_path = Join-Path $env:temp "OpenTelemetry.DotNet.Auto.psm1"
-   Invoke-WebRequest -Uri $module_url -OutFile $download_path -UseBasicParsing
-   ```
-
-   {{% /tab %}} {{< /tabpane >}}
-
-2. Execute following script to download automatic instrumentation for your
-   development environment:
-
-   {{< tabpane text=true >}} {{% tab Unix-shell %}}
-
-   ```sh
-   ./otel-dotnet-auto-install.sh
-   ```
-
-   {{% /tab %}} {{% tab PowerShell - Windows %}}
-
-   ```powershell
-   Import-Module $download_path
-   Install-OpenTelemetryCore
-   ```
-
-   {{% /tab %}} {{< /tabpane >}}
-
-3. Set and export variables that specify a [console exporter][], then execute
-   script configuring other necessary environment variables using a notation
-   suitable for your shell/terminal environment &mdash; we illustrate a notation
-   for bash-like shells and PowerShell:
+```shell
+dotnet add package OpenTelemetry.Extensions.Hosting
+dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
+dotnet add package OpenTelemetry.Exporter.Console
+```
```

Comment:
If you look at the preview, you'll see that the list item numbering is off: https://deploy-preview-3364--opentelemetry.netlify.app/docs/instrumentation/net/getting-started/#instrumentation

You need to indent the list-item contents, like this:

~~~suggestion
    ```shell
    dotnet add package OpenTelemetry.Extensions.Hosting
    dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
    dotnet add package OpenTelemetry.Exporter.Console
    ```
~~~

If you indent all list item content correctly, then Prettier will sequentially number all list items.